docs/units-and-global-variables.rst

```diff
@@ -75,7 +75,8 @@ Block and Transaction Properties
 - ``block.basefee`` (``uint``): current block's base fee (`EIP-3198 <https://eips.ethereum.org/EIPS/eip-3198>`_ and `EIP-1559 <https://eips.ethereum.org/EIPS/eip-1559>`_)
 - ``block.chainid`` (``uint``): current chain id
 - ``block.coinbase`` (``address payable``): current block miner's address
-- ``block.difficulty`` (``uint``): current block difficulty
+- ``block.difficulty`` (``uint``): current block difficulty (``EVM < Paris``). For other EVM versions it behaves as ``block.prevrandao`` (`EIP-4399 <https://eips.ethereum.org/EIPS/eip-4399>`_ )
+- ``block.prevrandao`` (``uint``): random number provided by the beacon chain (``EVM >= Paris``)
```

Comment:
While it's nice to have these next to each other, the list is alphabetic, isn't it?

Reply:
yes, I thought that was ordered only by the first word(i.e. `block`) and not by all. Changed it.